packages/components/src/Components/ConditionalFilter/ConditionalFilter.js

```diff
@@ -54,11 +54,15 @@ class ConditionalFilter extends Component {
                                         onSelect={ () => this.dropdownToggle(false) }
                                         isOpen={ isOpen }
                                         toggle={
-                                            <DropdownToggle onToggle={ this.dropdownToggle }>
+                                            <DropdownToggle
+                                                onToggle={ this.dropdownToggle }
+                                                className={ hideLabel ? 'no__label' : '' } >
```

Comment:
The className is a bit unfortunate. Can you name it perhaps `ins-c-conditional-filter__no-label`?